pkg/openstack/client/client.go

```diff
@@ -175,3 +175,20 @@ func IsNotFoundError(err error) bool {
 
 	return false
 }
+
+// IgnoreNotFoundError ignore not found error
+func IgnoreNotFoundError(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	if _, ok := err.(gophercloud.ErrDefault404); ok {
+		return nil
+	}
+
+	if _, ok := err.(gophercloud.Err404er); ok {
+		return nil
+	}
+
+	return err
```

Comment:
It does not make sense to duplicate the logic from `IsNotFoundError`. We can reuse it.

```suggestion
	if IsNotFoundError(err) {
		return nil
	}
	return err
```

Reply:
committed in 78d564d6